apps/prairielearn/src/components/SideNav.html.ts

```diff
@@ -213,7 +213,7 @@ function CourseNav({
       <div id="course-dropdown" class="dropdown">
         <button
           type="button"
-          class="btn dropdown-toggle dropdown-menu-right border border-gray bg-white w-100 d-flex justify-content-between align-items-center mb-2"
+          class="btn dropdown-toggle dropdown-menu-end border border-gray bg-white w-100 d-flex justify-content-between align-items-center mb-2"
```

Comment:
I'm curious why this wasn't caught by the compatibility tool.

Reply:
Ok, this wasn't flagged because `dropdown-menu-right` (or `-end`) does not apply if `dropdown-menu` itself is not in place. In essence, this class has no effect in this context because this is the dropdown trigger, not the menu. @miguelaenlle did you intend to have this menu right-aligned? If so, I can apply it to the menu div, otherwise I'll just remove this class.